### DISCUSSION TITLE
Access the router object

### DISCUSSION THREAD

**Discussion by eric-2d** [frandiox/vite-ssr#162] [Q&A] [answered]
Hi there,
In my main.js I need to pass the `router` object to a third party modules
https://github.com/gtm-support/vue-gtm#configuration

How would you go about that?
Thanks

**Comment by frandiox** [comment, marked as answer]
You can do that in the [main hook](https://github.com/frandiox/vite-ssr#usage). The router is passed inside the `context` object.

**Reply by eric-2d** [reply]
duh sorry, could have figured this one out.
Thanks mate :+1:
